### PR TITLE
Fixes for minor changes needed on code_of_conduct and young coders pages

### PR DESCRIPTION
**Pull request by bsdtux**
* Fixed code of conduct to now reference short_name in data/_config.yml
* Removed extra closing parenthesis in young coders
* Project now has a gemfile that contains required libraries for jekyll

### CODE REVIEW

File: about/diversity.md [hunk 1]
```diff
@@ -22,12 +22,12 @@ This is an ongoing process. We are talking to our program chairs, program commit
 
 Here are some ways you can help us build a more diverse conference experience:
 
-* Recommend appropriate speakers and/or program committee members to the conference chairs. You may also send an email to pycon-staff.
+* Recommend appropriate speakers and/or program committee members to the conference chairs. You may also send an email to pytn-staff.
```
Comment (epochblue) [inline about/diversity.md]: We should probably change these references to "pytn-staff" to "[organizers@pytennessee.org](mailto:organizers@pytennesee.org)". We don't have a `pytn-staff@` email address...

Comment (bsdtux) [reply about/diversity.md]: Sounds good. I can work on that tomorrow.

File: about/diversity.md [hunk 2]
```diff
@@ -22,12 +22,12 @@ This is an ongoing process. We are talking to our program chairs, program commit
 
 Here are some ways you can help us build a more diverse conference experience:
 
-* Recommend appropriate speakers and/or program committee members to the conference chairs. You may also send an email to pytn-staff.
+* Recommend appropriate speakers and/or program committee members to the conference chairs. You may also send an email to [organizers@pytennessee.org](mailto:{{site.data.event.organizers_email}}).
```
Comment (gtback) [inline about/diversity.md]: Should we use the variable for the display text as well as the mailto link?

Comment (bsdtux) [reply about/diversity.md]: I can try again but I was having issues with it rendering for some reason. It was like it didn't like the variable inside their

Comment (bsdtux) [reply about/diversity.md]: Huh, gave it the old college try and it let me do it this time. Not sure why it hated how I did it before

**Comment by epochblue** [comment]
Thank you for doing this, Josh!

**Pull request merged**